_posts/2022-09-30-one-million-enitities-in-one-minute.markdown

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title:  "Improving Anomaly Detection: One million entities in one minute"
+authors:
+- sudipto
+- naarcha
```

Comment:
Please add bpavani and joshtok...Two more authors still need to upload their author cards/photos